manifests/rhoai/kustomization.yaml

```diff
@@ -0,0 +1,47 @@
+# RHOAI configuration for Kubeflow Training Operator (KFTO)
+
+# Adds namespace to all resources.
+namespace: opendatahub
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kfto-
```

Comment:
@astefanutti, @tedhtchang and @jbusche Does this prefix make sense ?

Reply:
Based on the names from `base` I'd suggest `kubeflow-` as a prefix.

Reply:
Ok. will fix.